docs/optlib.rst

```diff
@@ -31,7 +31,7 @@ passed through command line.
 
 The following file is an example of an option file:
 
-.. code-block::
+.. code-block:: perl
```

Comment:
Yes, this is one of the acceptable choices.
.ctags file includes regular expressions. So specifying perl for highlighting the blocks is a pragmatic solution.